systems/plants/RigidBodyCylinder.m

```diff
@@ -103,6 +103,19 @@ function writeWRLShape(obj,fp,td)
       
       capsule = RigidBodyCapsule(obj.radius,obj.len,obj.T);
     end
+    
+    function h = draw(obj,model,kinsol,body_ind)
+      persistent cylinder_pts; % for all cylinders
+      if isempty(cylinder_pts)
+        [x,y,z] = cylinder;
+        cylinder_pts = [x(:)'; y(:)'; z(:)'-.5; 1+0*x(:)'];
+      end
+      
+      pts = forwardKin(model,kinsol,body_ind,obj.T(1:3,:)*diag([obj.radius,obj.radius,obj.len,1])*cylinder_pts);
+      N = size(pts,2)/2;
+      % todo: specify the color
+      h = surf([pts(1,1:N);pts(1,N+1:end)],[pts(2,1:N);pts(2,N+1:end)],[pts(3,1:N);pts(3,N+1:end)]);
```

Comment:
There may be a more efficient way to do this by updating the surf object rather than constructing a new one. This is certainly fine for now though.